### ISSUE TITLE
endless loading of Avito task list and some others

### ISSUE THREAD

**Issue by ilyadubrovsky** [31b4/Leetcode-Premium-Bypass#34]
**Describe the bug**
endless loading of Avito task list and some others

**To Reproduce**
https://leetcode.com/problemset/
find "Avito" in "Trending Companies"
click

**Screenshots**
![image](https://github.com/31b4/Leetcode-Premium-Bypass/assets/76615473/9deb26ef-8c0c-4dce-8ad1-019e572ca204)

**Comment by 31b4** [comment]
> **Describe the bug**
> 
> endless loading of Avito task list and some others
> 
> 
> 
> **To Reproduce**
> 
> Steps to reproduce the behavior:
> 
> https://leetcode.com/problemset/
> 
> find "Avito" in "Trending Companies"
> 
> click
> 
> 
> 
> **Screenshots**
> 
> ![image](https://github.com/31b4/Leetcode-Premium-Bypass/assets/76615473/9deb26ef-8c0c-4dce-8ad1-019e572ca204)

Yes sadly the database was last updated in October, i'll try to update it.

**Comment by Reidak18** [comment]
The problem is still relevant (checked for Avito)

**Comment by 31b4** [comment]
@Reidak18 i know, thats why the issue is open, i will try to update the database, it's just complicated.
